3lvia/kunde-elvia-web
# Feature/omega 120 power outage map v2

OMEGA-120

## What is the reason for this change?

Needs map component in poweroutage.

### What is the expected result?

Power outage map v2 should now show a simple map.

## How is it solved?

Uses `GoogleMap` component from `hdd-react`. Updated existing default center to point to Oslo instead of somewhere in the US.

## Any changes/updates that may affect others?

|                       | Changed |
| --------------------- | ------- |
| Unit tests            |        |
| Dependencies          |         |
| Environment variables |         |
| Build scripts         |         |

## How can it be tested and verified?
> IMPORTANT! Must have `POWEROUTAGE_V2` environment variable set to `true` to see the map (this will be changed in an upcoming PR).
- [ ] It should now show a simple map: 
![image](https://user-images.githubusercontent.com/42540861/93194069-f4cc7480-f747-11ea-8f98-9ff6ae3add84.png)

- [ ] The map should be centered at Oslo.
